specification-components.md

```diff
@@ -0,0 +1,107 @@
+# Reusing Descriptions
+
+The examples given so far have started to grow large because of the verbose nature of OpenAPI documents. This page introduces a mechanism to remove redundancy from the document by reusing portions of it.
+
+## The Components Object
+
+The [Components Object](https://spec.openapis.org/oas/v3.0.3#componentsObject),
+accessible through the `components` field in the root [OpenAPI Object](https://spec.openapis.org/oas/v3.0.3#oasObject), contains definitions for objects to be reused in other parts of the document.
+
+<figure style="text-align:center">
+  <img src="img/components-object.svg"/>
+  <figcaption>The OpenAPI Object is explained in the <a href="specification-structure.md">Structure of an OpenAPI Document</a> page.<br/>The Schema Object is explained in the <a href="specification-content.md">Content of Message Bodies</a> page.<br/>The Response Object is explained in the <a href="specification-paths.md">API Endpoints</a> page.<br/>The Parameter Object is explained in the <a href="specification-parameters.md">Parameters and Payload of an Operation</a> page.</figcaption>
+</figure>
+
+Most objects in the OAS can be replaced by a **reference** to a **component**, drastically reducing document size and maintenance (just like methods do in programming languages).
```

Comment:
Dunno if we want to start a style guide for terms but "OAS" always seems like a Technical thing to me, and if I am trying to be more user friendly I talk about "OpenAPI". Either way, seeing as OAS is an ancronym, "the OAS" seems a little redundant.

So either 
```suggestion
Most objects in OAS can be replaced by 
```

or 

```suggestion
Most objects in OpenAPI can be replaced by 
```

Reply:
The rule regarding "the" in front of acronyms seems to be related to whether you pronounce it as a word or as separate letters. Since I have never pronounced this acronym, nor heard it pronounced, I'll trust your opinion and drop the "the" :)

Regarding "OAS" vs "OpenAPI Specification" I swap them freely, to avoid repetition. I'll try to favor "OpenAPI" to be more friendly.